Exiled.Loader/TranslationManager.cs

```diff
@@ -0,0 +1,169 @@
+// -----------------------------------------------------------------------
+// <copyright file="TranslationManager.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Loader
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using Exiled.API.Extensions;
+    using Exiled.API.Features;
+    using Exiled.API.Interfaces;
+    using Exiled.Loader.Features.Configs;
+    using YamlDotNet.Core;
+    using YamlDotNet.Serialization;
+    using YamlDotNet.Serialization.NamingConventions;
+    using YamlDotNet.Serialization.NodeDeserializers;
+
+    /// <summary>
+    /// Used to handle plugin translations.
+    /// </summary>
+    public static class TranslationManager
+    {
+        /// <summary>
+        /// Gets the translation serializer.
+        /// </summary>
+        public static ISerializer Serializer { get; } = new SerializerBuilder()
```

Comment:
Since they're equal, you should either directly use the `ConfigManager` serializer and deserializer or move both of them in the `Loader` class to decouple them from the `ConfigManager`.